lib/services/presets_test.go

```diff
@@ -60,160 +69,187 @@ func TestAddRoleDefaults(t *testing.T) {
 			expected: &types.RoleV6{
 				Metadata: types.Metadata{
 					Name: teleport.PresetEditorRoleName,
+					Labels: map[string]string{
+						types.TeleportManagedLabel: types.IsManaged,
+					},
 				},
```

Comment:
for the existing roles, can we add test cases with/without the managed label already set?